common/src/chain/config/mod.rs

```diff
@@ -459,9 +460,14 @@ impl ChainConfig {
         self.data_deposit_min_fee
     }
 
-    /// The fee for issuing a token
-    pub fn token_min_issuance_fee(&self) -> Amount {
-        self.token_min_issuance_fee
+    /// The fee for issuing a fungible token
+    pub fn fungible_token_min_issuance_fee(&self) -> Amount {
```

Comment:
Btw, I think it's not appropriate to call them "min_xxxxx" anymore, because burning isn't explicit. Previously when we burned, burning more was OK, and hence this is "min" fee. Now since we're subtracting, this name doesn't make sense any more.

Please go through all the relevant "min" things and remove "min" from them.